plugin.go

```diff
@@ -3,13 +3,12 @@ package main
 import (
 	"bytes"
 	"fmt"
-
 	"os"
 	"os/exec"
+	"strings"
 
-	"path/filepath"
-
-	"github.com/drone-plugins/drone-git-push/repo"
+	"github.com/appleboy/drone-git-push/repo"
```

Comment:
Was this changed for any specific reason?

Reply:
you can see the https://github.com/drone-plugins/drone-git-push

redirect to https://github.com/appleboy/adrone-git-push

Reply:
No problem. What was the reason for the transfer?